docs/explanation/oci-image.md

```diff
@@ -0,0 +1,24 @@
+# OCI Image
+
+### wordpress-image
+
+The wordpress-image is custom built to include a default set of plugins and themes. The list of
+plugins and themes can be found at the reference section of the
+[documentation](https://charmhub.io/wordpress-k8s/docs/reference?channel=edge). Since WordPress is
+an application running on php, required libraries and dependencies are installed during the build
+process.
+
+WordPress application installation is done at runtime once during charm initialisation.
```

Comment:
Added the description to clarify that it is done during database setup.
```
WordPress application installation is done at runtime during database connection setup. This can
happen during database relation changed, database relation joined or database config changed
events.
```
Thank you for the clarification!